Mixpanel/MixpanelInstance.swift

```diff
@@ -1092,7 +1120,7 @@ extension MixpanelInstance: InAppNotificationsDelegate {
 
     // MARK: - Decide
     func checkDecide(forceFetch: Bool = false, completion: @escaping ((_ response: DecideResponse?) -> Void)) {
-        serialQueue.async {
+        trackingQueue.async {
```

Comment:
this should be part of the network queue but also wait for the tracking queue to finish its current tasks